Sources/AppcuesKit/AppcuesKit.docc/AnchoredTooltips.md

```diff
@@ -0,0 +1,51 @@
+
+## Instrumenting UIKit Views
+
+The following `UIView` properties are used to identify elements, in order of precedence:
+
+* [`accessibilityIdentifier`](https://developer.apple.com/documentation/uikit/uiaccessibilityidentification/1623132-accessibilityidentifier) - commonly used for Xcode UI testing, and generally a good practice to include on core views in the application to help identify.
+* [`tag`](https://developer.apple.com/documentation/uikit/uiview/1622493-tag) - an integer value that can be used to identify view objects in your application.
+* [`accessibilityLabel`](https://developer.apple.com/documentation/objectivec/nsobject/1615181-accessibilitylabel) - a property used in assistive applications, such as VoiceOver, to convey information to users with disabilities, to help them use the app. In addition to being a good practice to use to support core accessibility features on iOS, it can also help with Appcues element targeting. It is considered a lower priority property for selector usage, however, due to the fact that the descriptive strings can often be non-unique throughout the application and they may also be localized to the user's preferred language and not maintain consistency with the label used when the flow was built.
+
+At least one identifiable property must be set. Not all are required. The best way to ensure great performance of iOS anchored tooltips in Appcues is to set a unique `accessibilityIdentifier` on each `UIView` element that may be targeted.
+
+These properties are available on any UIView derived type, and can be set programmatically in Swift as follows:
+
+```swift
+let button = UIButton()
+button.accessibilityIdentifier = "btnSaveProfile"
+button.accessibilityLabel = "Save"
+button.tag = 1234
+```
+
+These properties are also available in Xcode Interface Builder, used for Storyboard UI development. Click on the view and select the Identity inspector tab to see the Accessibility section and set related properties. The `tag` is also visible to edit on the Attributes inspector. These properties can also be set using the User Defined Runtime Attributes section of the Identity inspector for a selected element in Interface Builder.
+
+> Tip: For the common use case of `UITabBarController` and its associated `UITabBar` items, the Appcues iOS SDK will automatically generate index-based selectors for each item to use for targeting tooltips. No additional code is required in your application if you would like to target the items in a `UITabBar` in your application's screen by index.
+
+## Instrumenting SwiftUI Views
+
+iOS Applications using SwiftUI can also identify `View` elements. A `View` extension `appcuesView(identifier:)` is provided by the Appcues iOS SDK to support this use case. The `identifier` String value must be unique on the screen where an anchored tooltip may be targeted.
```

Comment:
Was also unable to find a way to link to public API docs for this `appcuesView(identifier:)` extension. This is an extension method on `View`, and perhaps does not generate DocC output?

Reply:
It looks like Swift 5.9 and so maybe Xcode 14 might include the ability to do this, but it's not possible now.

https://forums.swift.org/t/enablement-of-docc-extension-support-as-an-opt-in-feature/62614/12